src/_lib/getTimezoneOffsetInMilliseconds/index.js

```diff
@@ -17,7 +17,7 @@ function getDateMillisecondsPart(date) {
  */
 export default function getTimezoneOffsetInMilliseconds(dirtyDate) {
   var date = new Date(dirtyDate.getTime())
-  var baseTimezoneOffset = Math.ceil(date.getTimezoneOffset())
+  var baseTimezoneOffset = Math.trunc(date.getTimezoneOffset())
```

Comment:
@kamranayub Not sure but looks like maybe [`Math.trunc` isn't supported in Internet Explorer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/trunc#browser_compatibility) - is IE support a requirement for this project? Maybe not - maybe consumers that need to support IE are able to polyfill that function and/or transpile this dependency with Babel `preset-env` or whatever.

Reply:
@kamranayub Maybe we could do something like this instead:

```suggestion
  var offset = date.getTimezoneOffset();
  var baseTimezoneOffset = offset < 0 ? Math.ceil(offset) : Math.floor(offset);
```

Reply:
The problem are values that aren't integers, which seems to be an issue in FF... they can still be positive or negative and have decimal digits that would need to be chopped off. There could be another approach that avoids using Math.trunc but yeah it seems core-js polyfills this.